system/Database/Query.php

```diff
@@ -473,6 +473,58 @@ protected function matchSimpleBinds(string $sql, array $binds, int $bindCount, i
 
 	//--------------------------------------------------------------------
 
+	/**
+	 * Returns string to display in debug toolbar
+	 *
+	 * @return string
+	 */
+	public function debugToolbarDisplay() : string
+	{
+		// Key words we want bolded
+		$highlight = [
```

Comment:
I don't know if there is a performance gain but maybe we can set this as `static` variable?
```suggestion
		static $highlight = [
```